alembic/versions/95a1fd846f59_initial_migration.py

```diff
@@ -0,0 +1,30 @@
+"""Initial migration
+
+Revision ID: 95a1fd846f59
+Revises: 307327a9d483
+Create Date: 2024-10-27 13:38:56.420725
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '95a1fd846f59'
+down_revision: Union[str, None] = '307327a9d483'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
```

Comment:
The `downgrade` function also does nothing (`pass`). Ensure this is intentional. If no changes are needed for downgrading, it's acceptable, but typically, you would include operations to revert the changes made in the `upgrade` function.